docs/source/index.rst

```diff
@@ -67,6 +67,23 @@ and how to run resultsdb ansible module:
 * with job_names provided: ``ansible <host> -m resultsdb -a "test_tier=1 nvr=name-version-release job_names=first-job,second-job resultsdb_api_url=resultsdb-url"``
 * without job_names: ``ansible <host> -m resultsdb -a "test_tier=1 nvr=name-version-release resultsdb_api_url=resultsdb-url"``
 
+Provision
++++++++++
+Provision plugin purpose is to create topology files. These files will be handled by linch pin tool.
+This plugin creates two files:
+* topology.json - contains all data needed for provisioning
+* topology_credentials.yaml - contains needed credentials for VM creation
+These two files are created from openstack config file.
```

Comment:
```
These two files are created from an opestack config file.
```
also might need to link to an openstack config url or something showing what that is.